dist/load-polyfills.js

```diff
@@ -1 +1,49 @@
-!function(){"use strict";function e(e){return function(t){var r=t.toLowerCase().match(e);return!!r&&parseInt(r[1],10)}}var t={chrome:e(/chrom(?:e|ium)\/([0-9]+)\./),firefox:e(/firefox\/([0-9]+\.*[0-9]*)/),safari:e(/version\/([0-9]+).[0-9]+.[0-9]+ safari/),ie:e(/(?:msie |rv:)([0-9]+).[0-9]+/),edge:e(/edge\/([0-9]+).[0-9]+.[0-9]+/)},r={firefox:[50,74],chrome:[50,80],ie:[6,11],edge:[12,80],safari:[9,13]};var i=function(){var e=navigator.userAgent,r="undefined"!=typeof InstallTrigger,i=/constructor/i.test(window.HTMLElement)||"[object SafariRemoteNotification]"===(!window.safari||window.safari.pushNotification).toString(),o=!!document.documentMode,a=!o&&!!window.StyleMedia,n=/HeadlessChrome/.test(e)||!!window.chrome&&!!window.chrome.loadTimes,s=/; wv/.test(e)&&/Chrome/.test(e),c=/SAMSUNG/.test(e)&&/Chrome/.test(e),f=n||s||c?"chrome":r?"firefox":i?"safari":o?"ie":!!a&&"edge";return{browser:f,version:!(!f||!t[f])&&t[f](e)}}(),o=i.browser,a=i.version,n=window.__DW_SVELTE_PROPS__.data.polyfillUri;o&&r[o]&&a>=r[o][0]?a<=r[o][1]&&document.write('<script type="text/javascript" src="'.concat(n,"/").concat(o,"-").concat(a,'.js"><\/script>')):document.write('<script type="text/javascript" src="'.concat(n,'/all.js"><\/script>'))}();
\ No newline at end of file
+(function(){'use strict';function e(e) {
```

Comment:
Not sure if this is important, but this file used to be minified but no longer is. Change seems unrelated to the rest of the PR (?)